huaweicloud/services/css/resource_huaweicloud_css_cluster.go

```diff
@@ -1443,7 +1443,16 @@ func updateKibanaPublicAccess(ctx context.Context, d *schema.ResourceData, cssV1
 
 	switch len(nValue) - len(oValue) {
 	case -1: // delete kibana_public_access
-		_, err := cssV1Client.UpdateCloseKibana(&model.UpdateCloseKibanaRequest{ClusterId: d.Id()})
+		_, err := cssV1Client.UpdateCloseKibana(&model.UpdateCloseKibanaRequest{
+			ClusterId: d.Id(),
+			Body: &model.CloseKibanaPublicReq{
+				EipSize: utils.Int32(int32(utils.PathSearch("bandwidth", oValue[0], 0).(int))),
+				ElbWhiteList: &model.StartKibanaPublicReqElbWhitelist{
+					EnableWhiteList: utils.PathSearch("whitelist_enabled", oValue[0], false).(bool),
+					WhiteList:       utils.PathSearch("whitelist", oValue[0], false).(string),
```

Comment:
The default value should be empty string
```suggestion
					WhiteList:       utils.PathSearch("whitelist", oValue[0], "").(string),
```